stories/components/breadcrumbs.stories.js

```diff
@@ -0,0 +1,8 @@
+import "../../sass/components/breadcrumbs.scss";
```

Comment:
You can remove this too-- I just put it in the `mdn-minimalist` version as a test and it wasn't using the updated breadcrumb styles either.